graylog2-web-interface/webpack.combined.config.js

```diff
@@ -11,9 +17,14 @@ const pluginConfigs = process.env.disable_plugins == 'true' ? [] : glob.sync(plu
 
 process.env.web_src_path = path.resolve(__dirname);
 
-module.exports = ['./webpack.config.js']
-  .concat(pluginConfigs)
-  .map(function(config) { return require(path.resolve(__dirname, config)); })
-  .reduce(function(config1, config2) {
-    return merge.smart(config2, config1);
-  });
+const webpackConfig = require(path.resolve(__dirname, './webpack.config.js'));
+
+pluginConfigs.forEach(function(plugin) {
+  const pluginName = plugin.split('/')[2];
+  const pluginDir = path.resolve(__dirname, '../..', pluginName, 'src/web');
```

Comment:
This needs to take into account that some plugins can have submodules, like above for `pluginPattern2`.